src/test/scala/com/databricks/spark/csv/CsvWriteSuite.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2014 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.databricks.spark.csv
+
+import java.io.File
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, FileUtil, Path}
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, SQLContext}
+import org.scalatest.{BeforeAndAfterAll, FunSuite}
+
+import scala.io.Source
+
+class CsvWriteSuite extends FunSuite with BeforeAndAfterAll {
+
+  val datesFile = "src/test/resources/dates.csv"
+  val tempEmptyDir = "target/test/empty/"
+
+  def parserLib: String = "COMMONS"
```

Comment:
I made the change, but I still do not agree that the tests for writing (which do not involve the parserLib other than to read a file so that it can test the writing of it) need to be run this way. Maybe I don't understand. Are you saying that the parserLib is used for writing in addition to reading? Given the name, I thought it would only be involved with reading.